src/stellar/stellarEngine.ts

```diff
@@ -67,12 +67,39 @@ export class StellarEngine extends CurrencyEngine<StellarTools> {
     this.pendingTransactionsMap = {}
     this.fees = { low: BASE_FEE, standard: BASE_FEE, high: BASE_FEE }
     this.minimumAddressBalance = this.networkInfo.baseReserve
+
+    this.getRecipientBalance = this.getRecipientBalance.bind(this)
   }
 
   setOtherData(raw: any): void {
     this.otherData = asStellarWalletOtherData(raw)
   }
 
+  async getRecipientBalance(recipient: string): Promise<string> {
```

Comment:
If you use an arrow function `getRecipientBalance = async () => {}`, you can drop the `bind` call in the constructor.